src/cleanfigure.m

```diff
@@ -939,6 +939,30 @@ function removeNaNs(meta, handle)
     topRight    = [xLim(2); yLim(2)];
 end
 % =========================================================================
+function data = limitPrecision(data)
+  % Limit the precision of the given data
+  
+  % Only do something if the data is not empty
+  if isempty(data) || all(~isfinite(data))
+      return
+  end
+  
+  % Only consider finite values
+  mask = isfinite(data);
+
+  % Get the maximal value of the data
+  range = max(abs(data(mask)));
```

Comment:
That's not a bad idea to avoid confusion. Maybe `maxVal` or `maxAbs` or something similar.

Also, I think both lines of code `mask = ...` and this one, can happen on a single line. It's nothing too complex.

Regarding the comments: I think we can do with fewer comments, given that the explain *what* is done (== job of the code) and not *why* it is done (== job of the comments).
If both lines are put on a single line, it makes a tiny bit more sense to put a comment to help reading the code
```matlab
% Determine the largest value that is still finite
largestValue = max(abs(data(isfinite(data))));
```